apps/scan/backend/src/app_export.test.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  getCastVoteRecordExportDirectoryPaths,
+  readCastVoteRecordExport,
+} from '@votingworks/backend';
+import { assertDefined, ok } from '@votingworks/basics';
+import { mockOf } from '@votingworks/test-utils';
+import { CVR } from '@votingworks/types';
+import {
+  BooleanEnvironmentVariableName,
+  convertCastVoteRecordVotesToTabulationVotes,
+  getCurrentSnapshot,
+  getFeatureFlagMock,
+} from '@votingworks/utils';
+
+import { scanBallot, withApp } from '../test/helpers/custom_helpers';
+import { configureApp } from '../test/helpers/shared_helpers';
+
+jest.setTimeout(30_000);
+
+const mockFeatureFlagger = getFeatureFlagMock();
+
+jest.mock('@votingworks/utils', (): typeof import('@votingworks/utils') => {
+  return {
+    ...jest.requireActual('@votingworks/utils'),
+    isFeatureFlagEnabled: (flag) => mockFeatureFlagger.isEnabled(flag),
+  };
+});
+
+beforeEach(() => {
+  mockFeatureFlagger.resetFeatureFlags();
+  mockFeatureFlagger.enableFeatureFlag(
+    BooleanEnvironmentVariableName.SKIP_ELECTION_PACKAGE_AUTHENTICATION
+  );
+});
+
+test('continuous CVR export, including polls closing', async () => {
```

Comment:
Copied these tests over, as is, from `app_config.test.ts` to `app_export.test.ts`. So no new code in this commit